docs/src/adv/logging.md

```diff
@@ -17,40 +17,41 @@ import Term: install_term_logger
 install_term_logger()
 ```
 
-!!! note
-    We could not get Documenter.jl to show the output of `Term`'s logger. So we've added a screenshot at the bottom of this page!
-
 Done. Not a lot of work. Now you can just use your logging macros as you would normally, you don't need to change anything. But magic will happen:
 
-```Julia
-import Term: install_term_logger # hide
-install_term_logger() # hide
+```@example termlogger
+import Term: TermLogger, theme; import Logging: with_logger # hide
+with_logger(TermLogger(stderr, theme)) do # hide
+
 @info "My log message" 1+1 n="a string!" :x
+
+end # hide
 ```
 
 As you can see, it shows all the information that is normally there, an more! 
 If your log macro is called from within a function, it will also tell you that:
```

Comment:
Btw, I noticed that this doesn't seem to work as in your screenshot (not outside of Documenter either).